radixdlt-java/src/main/java/com/radixdlt/client/core/network/RadixNetwork.java

```diff
@@ -25,28 +24,41 @@
 	/**
 	 * Hot observable which updates subscribers of new connection events
 	 */
-	private final ConnectableObservable<SimpleImmutableEntry<String, RadixClientStatus>> statusUpdates;
-
+	private final ConnectableObservable<SimpleImmutableEntry<RadixPeer, RadixClientStatus>> statusUpdates;
```

Comment:
RadixPeer isn't immutable so weird to have it as a key.

Reply:
This isn't strictly a key as it's just a single entry. For clarity I've changed it to use Pair instead.